.github/workflows/test-package.yml

```diff
@@ -67,8 +81,20 @@ jobs:
           npm install --global yarn
           echo "yarn: $(yarn --version)"
 
+      - name: Set dependency overrides
+        env:
+          SET_OVERRIDES_COMMAND: ${{ inputs.set-overrides-command }}
+          STYLELINT_VERSION_OVERRIDE: ${{ inputs.stylelint-version-override }}
+        if: ${{ inputs.set-overrides-command }}
+        run: |
+          if ! eval "$SET_OVERRIDES_COMMAND=$STYLELINT_VERSION_OVERRIDE"; then
+            echo "::error ::Setting dependency overrides failed."
+            exit 1
+          fi
+
+          npm pkg get;
```

Comment:
This logs the `package.json` contents, allowing us to inspect the values for debugging purposes.